tests/features/editorial_workflow.feature

```diff
@@ -96,3 +96,12 @@ Feature: Editorial workflow
       | editor             |
       | contributor        |
       | authenticated user |
+
+  Scenario: User role "editorial team member" cannot be assigned via the UI.
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/people"
+    Then I should not have the following options for "edit-operation":
```

Comment:
This function only checks on the first level of the option ("editorial team member" is in a second level of options)
It doesn't check in "optgroup".